conf/views/secondaryContactTelephoneNumber.njk

```diff
@@ -0,0 +1,58 @@
+{% extends "includes/layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"     import govukBackLink %}
+{% from "govuk/components/input/macro.njk"         import govukInput %}
+{% from "govuk/components/button/macro.njk"        import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("secondaryContactTelephoneNumber.title"), form.errors) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  {{ govukBackLink({
+    text: messages("site.back")
+  }) }}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% if form.errors.length > 0 %}
+          {{ govukErrorSummary({
+            "titleText": messages("error.summary.title"),
+            "errorList": form.errors
+          }) }}
+        {% endif %}
+
+        <form action="{{ routes.controllers.SecondaryContactTelephoneNumberController.onSubmit(mode).url }}" method="post">
+
+          {{ csrf() | safe }}
+
+          {{ govukInput({
+            label: {
+              text: messages('secondaryContactTelephoneNumber.heading', contactName),
+              isPageHeading: true,
+              classes: "govuk-label--xl"
+            },
+            id: "value",
+            name: "value",
+            hint: {text: messages('$className;format="decap"$.hint')},
```

Comment:
Change to messages('secondaryContactTelephoneNumber.hint') to display hint message